### PR TITLE
feat(popover): add a CSS prop for the max height of the popover

### PR DESCRIPTION
**Pull request by Kiarokh**
fix https://github.com/Lundalogik/crm-feature/issues/3475

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/popover-surface/popover-surface.scss [hunk 1]
```diff
@@ -52,4 +52,14 @@
...
     }
+
+    & > * {
+        // this fixes some unwanted layout issues
+        // which might be very hard for consumers to realize
+        // and fix by themselves. Without this code,
+        // if the consumer sets for example paddings in
+        // what they send to the popover, their element
+        // can grow larger than the popover surface
+        box-sizing: border-box;
```
Comment (Kiarokh) [inline src/components/popover-surface/popover-surface.scss:63]: image
In this example, the component inside the popover doesn't have the `box-sizing: border-box` and a part of it is being pushed out, due to the paddings.

File: src/components/popover-surface/popover-surface.scss [hunk 2]
```diff
@@ -3,18 +3,18 @@
...
-}
 
-.limel-popover-surface {
     position: relative;
     display: flex;
-    flex-direction: column;
 
     width: var(--popover-surface-width, auto);
-    max-height: calc(100vh - 2rem);
+    max-height: inherit; // inherits it from the dynamically calculated `max-height` of `limel-portal--container`
```
Comment (Kiarokh) [inline src/components/popover-surface/popover-surface.scss:11]: Screenshot 2023-06-20 at 16 32 02

File: src/components/popover-surface/popover-surface.scss (hunk 1 above)
Comment (Befkadu1) [inline src/components/popover-surface/popover-surface.scss:63]: How did you reproduce this?

Comment (Befkadu1) [reply src/components/popover-surface/popover-surface.scss:63]: How did you reproduce this?

**Review by Befkadu1 — APPROVED** [review]

**Pull request merged**